### DISCUSSION TITLE
EV3 crashes after a while, and when rebooting the brick, normal EV3 environment runs.

### DISCUSSION THREAD

**Discussion by alibustami** [pybricks/support#296] [Q&A]
After imaging the EV3 MircoPython environment V2 by Etcher (1.5.51), some times EV3 brick doesn't recognize SD card and runs normal EV3 environment, and sometimes after running some programs suddenly bricks crushes and stops responding and then after force rebooting, again runs the normal EV3 environment.

**Comment by dlech** [comment]
The SD card might not be making good contact. Try removing it and putting it back in again. If that doesn't work, try a different SD card. If that doesn't work, the SD card slot on the EV3 may be defective. If that is determined to be the case, contact LEGO customer support.
